### ISSUE TITLE
Cannot install on Fedora

### ISSUE THREAD

**Issue by gappleto97** [JoyStream/joystream-desktop#957]
It would be nice if you would provide an rpm, or a compressed binary

**Comment by mnaamani** [comment]
Done in #961 but will require more customization. Options available here: https://github.com/unindented/electron-installer-redhat#options

**Comment by mnaamani** [comment]
Latest release now builds an rpm package.

https://github.com/JoyStream/joystream-desktop/releases

Give it a try and let us know how well it works.

**Comment by gappleto97** [comment]
Gave it a shot, and I think you're having a collision with nvm stuff. I'm going to try removing that from my bashrc, so I'll get back to you on that in a minute or two

**Comment by gappleto97** [comment]
Yeah, this rpm doesn't seem to work. It's trying to link to `/usr/share/joystream/joystream`, but no such file exists when I install. I think you mean to be linking to `/usr/share/joystream/Joystream`, because running that seems to work (so far).

**Comment by mnaamani** [comment]
Thanks for the update @gappleto97 
could you give some additional information about how the install was colliding with `nvm`?

**Comment by gappleto97** [comment]
It wasn't. It was that linking error above that led me to believe it, but when disabling nvm it still happened, so I looked further. Its just the capitalization issue above.
